packages/contracts-bedrock/test/Safe/DeployOwnership.t.sol

```diff
@@ -88,5 +88,8 @@ contract DeployOwnershipTest is Test, DeployOwnership {
         assertEq(LivenessModule(livenessModule).livenessInterval(), lmConfig.livenessInterval);
         assertEq(LivenessModule(livenessModule).thresholdPercentage(), lmConfig.thresholdPercentage);
         assertEq(LivenessModule(livenessModule).minOwners(), lmConfig.minOwners);
+
+        // Ensure the threshold on the safe agrees with the LivenessModule's required threshold
+        assertEq(securityCouncilSafe.getThreshold(), LivenessModule(livenessModule).getRequiredThreshold(owners.length));
```

Comment:
Check isn't equivalent to the old one:
```solidity
_safe.getThreshold() >= getRequiredThreshold(owners.length)
```

Intentional?

Reply:
You mean because it was `gte`? 
That's correct. The previous check left some wiggle room for a more general usage of the module, but given the intention of this script as demonstrating the correct setup, I'm being more strict.